README.md

```diff
@@ -33,7 +33,7 @@ For complete Knative Eventing documentation, see
 
 If you are interested in contributing, see [CONTRIBUTING.md](./CONTRIBUTING.md),
 [DEVELOPMENT.md](./DEVELOPMENT.md) and
-[Knative WORKING-GROUPS.md](https://www.knative.dev/contributing/working-groups/#eventing).
+[Knative WORKING-GROUPS.md](https://knative.dev/community/contributing/working-groups/working-groups/#eventing).
```

Comment:
While we're at it, now is probably a good time to rename the link to "Knative working groups", since it doesn't point to a local .MD file anymore.

Reply:
Done.